src/sbvr-api/sbvr-utils.ts

```diff
@@ -1001,8 +1003,6 @@ export type Passthrough = AnyObject & {
 };
 
 export class PinejsClient extends PinejsClientCore {
-	// @ts-expect-error This is actually assigned by `super` so it is always declared but that isn't detected here
-	public passthrough: Passthrough;
```

Comment:
I suppose this is what was breaking target: es2022?

Reply:
Yes